libraries/SITL/SIM_GPS.h

```diff
@@ -327,16 +96,30 @@ class GPS : public SerialDevice {
 
     enum Type {
         NONE  =  0,
+#if AP_SIM_GPS_UBLOX_ENABLED
         UBLOX =  1,
+#endif
+#if AP_SIM_GPS_NMEA_ENABLED
         NMEA  =  5,
+#endif
+#if AP_SIM_GPS_SBP_ENABLED
         SBP   =  6,
+#endif
 #if AP_SIM_GPS_FILE_ENABLED
         FILE  =  7,
 #endif
+#if AP_SIM_GPS_NOVA_ENABLED
         NOVA  =  8,
+#endif
+#if AP_SIM_GPS_SBP2_ENABLED
         SBP2  =  9,
-        GSOF  = 11, // matches GPS_TYPE
+#endif
+#if AP_SIM_GPS_TRIMBLE_ENABLED
+        TRIMBLE  = 11, // matches GPS_TYPE
```

Comment:
Trimble sells multiple types of GPS; one of them communicates GSOF protocol. 

I'd propose calling it `TRIMBLE_GSOF` that way it doesn't conflict with others like `TRIMBLE_TSIP` which is another protocol used in the larger GNSS systems they sell.